user-guide/Cloud_Platform/TeamsBot/tutorials/ChatOps_Tutorials_Chat_Notification.md

```diff
@@ -19,39 +19,62 @@ Estimated duration: 15 minutes.
+- [Step 1: Add the DataMiner app to Microsoft Teams](#step-1-add-the-dataminer-app-to-microsoft-teams)
+- [Step 2: Deploy the ChatIntegration Examples package from the Catalog](#step-2-deploy-the-chat-integration-examples-package-from-the-catalog)
+- [Step 3: Fetch the private chat for a user and send a notification](#step-3-fetch-the-private-chat-for-a-user-and-send-a-notification)
+- [Step 4: Configure a Correlation rule triggered by a specific alarm to send the chat notification](#step-4-configure-a-correlation-rule-triggered-by-a-specific-alarm-to-send-the-chat-notification)
 
-## Step 1: Deploy the Chat Integration Examples package from the Catalog
+## Step 1: Add the DataMiner app to Microsoft Teams
 
-Learn more about this package [here](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples/README.md).
+If you have not yet installed the DataMiner Teams bot, follow the steps below. Otherwise, you can skip this step.
 
-1. Browse to <https://catalog.dataminer.services/details/package/5860>.
+1. In *Microsoft Teams*, go to *Apps*, and search for "DataMiner".
+
+1. Add the *DataMiner* app.
+
+> [!TIP]
+> For more detailed information, see [DataMiner Teams bot installation](xref:DataMiner_Teams_bot#dataminer-teams-bot-installation)
+
+## Step 2: Deploy the Chat Integration Examples package from the Catalog
+
+> [!NOTE]
+> To learn more about this package, refer to its [README file](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples/README.md).
+
+1. Go to <https://catalog.dataminer.services/details/package/5860>.
 
    ![ChatIntegration Examples page in Catalog](~/user-guide/images/chatops_notification_01_001.png)
 
 1. Click *Deploy* and choose your target DMS.
 
-1. Verify that the package was correctly installed by accessing the scripts in DataMiner Cube.
+1. Verify that the package has been installed correctly by accessing the scripts in the Automation module in DataMiner Cube.
 
    ![ChatIntegration Examples scripts](~/user-guide/images/chatops_notification_01_002.png)
 
-## Step 2: Fetch the private chat for a user and send a notification
+## Step 3: Fetch the private chat for a user and send a notification
 
-1. Execute the *Fetch Private Chat Example* script with the email address of the user.
+1. In the Automation module, select the *Fetch Private Chat Example* script, and click *Execute*.
+
+1. Specify the email address of the user and click *Execute now*.
 
    ![ChatIntegration Examples Fetch Private Chat](~/user-guide/images/chatops_notification_02_001.png)
 
-1. Verify that an entry was added in the *ChatsExample* memory file containing the private chat ID.
+1. Click *Close* to exit the *Fetch Private Chat Example* window.
+
+1. Verify that an entry has been added in the *ChatsExample* memory file containing the private chat ID.
 
    ![ChatIntegration Examples ChatsExample Memory file](~/user-guide/images/chatops_notification_02_002.png)
```

Comment:
not an issue for me, but the images were named with the first two digits being the step in the kata, maybe it's worth to adjust them accordingly?

Reply:
This is not really necessary – it's not worth the additional effort, since hardly anyone will notice or be bothered by this file name. 😊